src/main/scala/org/constellation/storage/SnapshotService.scala

```diff
@@ -390,7 +388,7 @@ class SnapshotService[F[_]: Concurrent](
       .map(_.snapshot)
       .flatMap { currentSnapshot =>
         if (currentSnapshot == Snapshot.snapshotZero) EitherT.rightT[F, SnapshotError](())
-        else write(currentSnapshot)
+        else write(currentSnapshot) >> EitherT.liftF(LiftIO[F].liftIO(dao.redownloadService.a(RecentSnapshot(currentSnapshot.hash, height, Map()))))
```

Comment:
Please use descriptive method names. `a` says nothing to me ;p

Reply:
I think I accidentally deleted that right before I pushed, should be persistLocalSnapshot